tests/run-build.sh

```diff
@@ -2,69 +2,68 @@
 
 set -ex
 
-# Skip build if requested
-if test -e ci-skip ; then
+function cleanup {
+    if [[ "${POSTCLEANUP}" == true ]]; then
+        for mp in $(mount | grep rkt | awk '{print $3}' | tac); do
+            sudo umount "${mp}"
+        done
+
+        for link in $(ip link | grep rkt | cut -d':' -f2); do
+            sudo ip link del "${link}"
+        done
+        sudo rm -rf /var/lib/cni/networks/*
```

Comment:
Or I can add additional parameter into script for forcing this behavior (or both). So code should be something like this
``` if [[ ("${POSTCLEANUP}" == true)  && (("{$CI}" == true) || ("${FORCE}" == true))); then```